### ISSUE TITLE
Default Qt download fails (Hunter 0.23.253). Qt 5.11.* archive repository is not existing anymore

### ISSUE THREAD

**Issue by zancos** [cpp-pm/hunter#191]
* I've read [Brief overview section](https://hunter.readthedocs.io/en/latest/overview.html) and do understand basic concepts. **[Yes]**
* I've read [F.A.Q. section](https://hunter.readthedocs.io/en/latest/faq.html) and there is no solution to my problem there. **[Yes]**
* I've read [Code of Conduct](https://github.com/cpp-pm/hunter/blob/master/.github/CODE_OF_CONDUCT.md), I promise to be polite and will do my best at being constructive. **[Yes]**
* I've read [Reporting bugs section](https://hunter.readthedocs.io/en/latest/contributing.html#reporting-bugs) carefully. **[Yes]**
* I've checked that all the `hunter_add_package`/`find_package` API used by me in the example is the same as in [documentation](https://hunter.readthedocs.io/en/latest/packages.html). **[Yes]**
* I'm using [latest](https://github.com/cpp-pm/hunter/releases) Hunter `URL`/`SHA1`. **[Yes, using FetchContent with https://github.com/cpp-pm/gate]**

* I've created [SSCCE](http://www.sscce.org/) reproducing the issue:
```cmake
# CMakeLists.txt

cmake_minimum_required(VERSION 3.5)

set(HUNTER_Qt_COMPONENTS qtbase)
set(HUNTER_PACKAGES Qt)


include(FetchContent)
FetchContent_Declare(SetupHunter GIT_REPOSITORY https://github.com/cpp-pm/gate)
FetchContent_MakeAvailable(SetupHunter)


project(MyProject LANGUAGES C CXX)
add_definitions(-DUNICODE)

set(CMAKE_INCLUDE_CURRENT_DIR ON)

set(CMAKE_AUTOUIC ON)
set(CMAKE_AUTOMOC ON)
set(CMAKE_AUTORCC ON)

set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)

find_package(Qt5 COMPONENTS Widgets LinguistTools REQUIRED)

...
```

Qt source repository for versions 5.11.* is not existing anymore.

[https://download.qt.io/archive/qt/](https://download.qt.io/archive/qt/)

Current Hunter version request versions 5.11 as default, producing download error.

**Comment by rbsheth** [comment]
This happened before - I think we have to change to `archive` URLs

**Comment by zancos** [comment]
Well, actually if you ask for 5.9, the link (witch is not `Qt archive`) works. The problem is, as I reported, that `Qt archive` removed links for version 5.11.x

**Comment by zancos** [comment]
> This happened before - I think we have to change to `archive` URLs

Well, actually if you ask for 5.9, the link (witch is not `Qt archive`) works. The problem is, as I reported, that `Qt archive` removed links for version 5.11.x

**Comment by truedat101** [comment]
5.10 is gone also.  Why is it closed if not fixed?

**Comment by NeroBurner** [comment]
fixed with #575
